tools/keycloak-user-initializer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+#############################################################################################
+###              Stage where Docker is building spring boot app using maven               ###
+#############################################################################################
+FROM maven:3.8.6-jdk-11 as build
+
+WORKDIR /opt/keycloak-init
+COPY . /opt/keycloak-init
+
+RUN mvn clean package -Dmaven.test.skip=true 
+
+RUN chown -R 1001:0 /opt/keycloak-init && \
+    chown -R 1001:0 $HOME && \
+    chmod -R ug+rwX /opt/keycloak-init
+
+USER 1001
+
+CMD ["java", "-jar", "/opt/keycloak-init/target/keycloak-user-initializer-0.0.1-SNAPSHOT.jar"]
```

Comment:
This is just a suggestion, but if you added a `<build><finalName></finalName></build>` property in your pom.xml, you could customize the name if the generated jar file without needing to update this Dockerfile every time the version changed in the pom.xml.
ie. https://github.com/bcgov/jag-efax/blob/main/jag-efax/pom.xml#LL164C1-L165C50
Then, add -Dtarget.fileName to the mvn command to specify the name of the jar at build time.

Reply:
That's a good suggestion. I will add a specific build name and update the Dockerfile as the way you suggested.